adafruit_sht4x.py

```diff
@@ -58,7 +64,7 @@ def add_values(cls, value_tuples):
             cls.delay[value] = delay
 
     @classmethod
-    def is_valid(cls, value):
+    def is_valid(cls, value) -> bool:
```

Comment:
See https://github.com/adafruit/Adafruit_CircuitPython_TLA202x/blob/main/adafruit_tla202x/__init__.py for help annotating `CV.is_valid()`, as they work identically.